package.json

```diff
@@ -1,94 +1,68 @@
   "scripts": {
-    "ng": "ng",
-    "start": "ng serve -o",
-    "build": "ng build",
-    "build-prod": "ng build --prod --build-optimizer",
     "build-lib": "ng-packagr -p lib",
-    "build-all": "rm -rf dist && npm run build-setup && npm run build-prod",
-    "build-setup":
-      "npm run build-lib && npm install dist/ngrx-data --no-save --no-package-lock",
-    "build-publish": "npm run build-lib && npm publish dist/ngrx-data",
-    "sme": "./node_modules/.bin/source-map-explorer",
-    "gen-stats": "ng build --prod --stats-json",
-    "gen-report": "webpack-bundle-analyzer dist/publicweb/stats.json",
-    "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e",
-    "docker-up": "docker-compose up -d --build",
-    "docker-up-debug":
-      "docker-compose -f docker-compose.debug.yml up -d --build",
-    "docker-down": "docker-compose down",
-    "dev":
-      "concurrently \"npm run debug\" \"ng serve --proxy-config proxy.conf.json --open\"",
-    "start-server": "node ./src/server/index.js",
-    "debug":
-      "PUBLICWEB=./dist/publicweb node -r dotenv/config --inspect src/server/index.js",
-    "serve-dist":
-      "PUBLICWEB=./dist/publicweb node -r dotenv/config --inspect dist/index.js",
-    "snyk-protect": "snyk protect",
-    "prepare": "npm run snyk-protect",
-    "precommit": "pretty-quick --staged"
+    "build-setup": "npm run build-lib && npm install dist/ngrx-data --no-save --no-package-lock"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^5.2.10",
-    "@angular/cdk": "^5.2.4",
-    "@angular/common": "^5.2.10",
-    "@angular/compiler": "^5.2.10",
-    "@angular/core": "^5.2.10",
-    "@angular/forms": "^5.2.10",
-    "@angular/http": "^5.2.10",
-    "@angular/material": "^5.2.4",
-    "@angular/platform-browser": "^5.2.10",
-    "@angular/platform-browser-dynamic": "^5.2.10",
-    "@angular/router": "^5.2.10",
-    "@ngrx/effects": "^5.1.0",
-    "@ngrx/entity": "^5.1.0",
-    "@ngrx/store": "^5.1.0",
-    "angular-in-memory-web-api": "^0.5.4",
+    "@angular/animations": "6.0.0",
+    "@angular/cdk": "^6.0.0",
+    "@angular/common": "6.0.0",
+    "@angular/compiler": "6.0.0",
+    "@angular/core": "6.0.0",
+    "@angular/forms": "6.0.0",
+    "@angular/http": "6.0.0",
+    "@angular/material": "^6.0.0",
+    "@angular/platform-browser": "6.0.0",
+    "@angular/platform-browser-dynamic": "6.0.0",
+    "@angular/router": "6.0.0",
+    "@ngrx/effects": "^6.0.0-beta.1",
+    "@ngrx/entity": "^6.0.0-beta.1",
+    "@ngrx/store": "^6.0.0-beta.1",
+    "angular-in-memory-web-api": "^0.6.0",
     "body-parser": "^1.18.2",
-    "core-js": "^2.4.1",
-    "dotenv": "^4.0.0",
-    "express": "^4.16.0",
+    "core-js": "^2.5.4",
+    "dotenv": "^5.0.1",
+    "express": "^4.16.3",
     "hammerjs": "^2.0.8",
     "handlebars": "^4.0.11",
-    "mongoose": "~4.11.1",
-    "rxjs": "^5.5.2",
-    "zone.js": "^0.8.14"
+    "mongoose": "^5.0.12",
+    "rxjs": "^6.1.0",
+    "zone.js": "^0.8.26"
```

Comment:
@coderabit-ai update package.json with the latest version of these dependencies.